packages/base/card-api.gts

```diff
@@ -966,7 +966,11 @@ class LinksTo<CardT extends CardDefConstructor> implements Field<CardT> {
       cachedInstance[isSavedInstance] = true;
       return cachedInstance as BaseInstanceType<CardT>;
     }
-    let resourceId = new URL(value.links.self, relativeTo).href;
+    let relativeId =
+      value.data && Array.isArray(value.data)
```

Comment:
this is a LinksTo remationship, not a LinksToMany, why would the value here ever be an array (keep in mind `value` here is the JSONAPI serialization for the relationship).